bin/kano-install-pip-deps

```diff
@@ -1,70 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Where were we even using this script?

Reply:
It was used by `peldins/stage-kano-jail.sh`, that's all I know :)

Reply:
Looks like it: https://github.com/search?q=kano-install-pip-deps&ref=simplesearch&type=Code&utf8=%E2%9C%93

I'll fix the QA test

Reply:
Ah, gotcha. Glad to see it go then :)